src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/AbiphoneFavourites.java

```diff
@@ -50,6 +50,7 @@ public class AbiphoneFavourites {
 
 	private static final AbiphoneFavourites INSTANCE = new AbiphoneFavourites();
 	private long lastClick = 0L;
+	private ItemStack checkShowSlot = null;
```

Comment:
Since the item stack is only ever used for one specific boolean check, a boolean could be used here.
Also: the name is somewhat lacking. Consider maybe `isInShowMenu`.